app/jobs/statcan_sync_job.rb

```diff
@@ -0,0 +1,13 @@
+class StatcanSyncJob < ApplicationJob
+  queue_as :default
+
+  def perform(statcan_dataset_id)
```

Comment:
nit: Using an id here is fine (and more space-efficient especially when using sidekiq or a redis-backed) but practically, it's nicer to let rails do it's magic with [global_id](https://github.com/rails/globalid) and just pass in the object. 

So instead when it's in the queue the argument looks like: `gid://OutcomeTrackerAPI/StatcanDataset/<id>` instead of `id`. This is nice from an operational perspective because you know what you're working with instead of just having an integer.